server/app/services/export/CsvExporter.java

```diff
@@ -22,59 +21,39 @@
  * {@link Path} indexing into an applicant's data, and CsvExporter takes the path and reads the
  * answer from {@link ReadOnlyApplicantProgramService} if present.
  */
-public class CsvExporter {
-  public static final CSVFormat DEFAULT_CSV_FORMAT =
```

Comment:
Note to the reviewer: This was unnecessarily shared between the implementation and the test.